### PR TITLE
[slackana-40] - [Markup] Create Project Overview Page

### PR DESCRIPTION
**Pull request by johnpaul-sun**
## Issue Link
-  https://app.asana.com/0/1203011167276287/1203026696603983/f

## Definition of Done
- [x] User can add team
- [x] User cannot delete the team if it only has one left
- [x]  User can close the modal
- [x] User can open the modal when clicking the add team button

## Notes 
- (Optional)

## Pre-condition
- yarn

## Expected Output
- The add team modal should pop up when the user clicks the add team button
- The add team modal should close when the user clicks outside of the modal or clicks the close button
- It should disable the team when there's only one left
- It should add a team if the user inputs teams in input tags

## Screenshots/Recordings
- Add team modal
image
- Disabled
image

### CODE REVIEW

File: client/src/components/molecules/TeamTemplate/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+              )}
+            </Menu.Item>
+          </div>
+        </Menu.Items>
+      </Transition>
+    </Menu>
+  );
+};
+
+export default TeamTemplate;
```
Comment (jsvelte) [inline client/src/components/molecules/TeamTemplate/index.tsx]: Add new line

**Review by jsvelte — APPROVED** [review]
# Approved 💯

File: client/src/shared/jsons/membersData.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    name: "Roger Gouse",
+    isActive: false,
+    team: "Backend"
+  },
+  {
+    name: "Roger Gouse",
+    isActive: false,
+    team: "Backend"
+  }
+]
```
Comment (abduljalilpalala) [inline client/src/shared/jsons/membersData.ts]: Add new line

File: client/src/shared/jsons/teamTempoData.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  },
+  {
+    team: 'Quality Assurance',
+    icon: '/images/team/qa.png',
+  },
+  {
+    team: 'Designer',
+    icon: '/images/team/designer.png',
+  },
+];
```
Comment (abduljalilpalala) [inline client/src/shared/jsons/teamTempoData.ts]: Add new line

**Review by abduljalilpalala — CHANGES_REQUESTED** [review]
Check comments.

**Review by abduljalilpalala — APPROVED** [review]
Check comments.

**Review by ipilikent — COMMENTED** [review]
## Approved PR
-  All displayed is the same as the Figma design
-  The modal is displayed
-  The Project Overview is responsive

## Proof
<details>

Project Overview


</details>

Test Case : https://docs.google.com/spreadsheets/d/1pa_tZVA7ar1JsgKeIS_0JTaJX9ZywKrX/edit?pli=1#gid=421111680
-----
- Total Test time: 1.8 hrs
- Total Comments: 0
- Total Iterations: 0

**Pull request merged**